docs/docs/userguide/monitoring/PRE_CANNED_CHARTS.md

```diff
@@ -0,0 +1,55 @@
+# Pre-canned product charts
+
+The instructions outlined on this page provide details on how you can deploy a set of Grafana dashboards specific to your provisioned DC product(s)
+
+!!!tip "Pre-requisites"
+    JMX metrics, Prometheus and Grafana will all need to be provisioned already to make use of this guide. For instructions on how to do this see, [Monitoring with Prometheus](PROMETHEUS.md)
+
+## 1. Enable Pre-canned dashboards
+
+Update the `grafana` stanza within the deployments `values.yaml` to enable dashboard creation with the appropriate labels i.e.
+
+```yaml
+monitoring:
+  grafana:
+    createDashboards: true
+    dashboardLabels:
+      grafana_dashboard: "1"
```

Comment:
If we override the default label in https://github.com/atlassian/data-center-helm-charts/pull/572/files#diff-04ac5986a1aff6b38d3846eb82f91416c2a3a8bee39cbb43a259237fa3132607R35 then we need to change it here as well, or we delete it from custom values yaml for monitoring stack (just to keep our examples consistent)

Reply:
For consistency, Ill change it [here](https://github.com/atlassian/data-center-helm-charts/pull/572/files#diff-04ac5986a1aff6b38d3846eb82f91416c2a3a8bee39cbb43a259237fa3132607R35) to `1` too as this is the default.